### PR TITLE
Restructuring coiote

### PR DESCRIPTION
**Pull request by laurensslats**
Big restructuring:

- Restructured Coiote IoT DM section:
  - Changed name from _Quick Start_ to _Introduction_ (as we already have a Quick Start under _Getting Started_)
  - Split Onboarding into: _Connect Devices_, _Certificate Mode_ & _Bootstrap Server_
  - Merged _Dashboard_ & _Data visualization_
  - Merged _Quickstart_ & _Device Onboarding_
  - Moved _Device Center_ to folder _Introduction_
  - Got rid of folder _User Interface_ (IMO no-one searches for _User Interface_)
- Improved a bunch of images
- Rearranged demo projects
- Added a bunch of redirects

TBD in next PR
- Link Getting Started docs to Device Onboarding doc

### CODE REVIEW

File: docs/Coiote_IoT_DM/introduction.md [hunk 1]
```diff
@@ -1,31 +1,70 @@
...
 
-## Device Groups (paid version only)
+Use the Operations Center to easily monitor your devices. It provides a clear overview with alerts, highlighting what devices require immediate attention.  
+
+![Operations Center](images/operations-center.png "Operations Center")
+
+## Device Groups*
+
+!!! Note
+    Some features indicated with an ***** are only included in the premium version of Coiote.
```
Comment (malw4) [inline docs/Coiote_IoT_DM/introduction.md:41]: Maybe the ***** should be in quotation marks?
```suggestion
    Some features indicated with an "*****" are only included in the premium version of Coiote.
```

File: docs/Coiote_IoT_DM/introduction.md [hunk 2]
```diff
@@ -1,31 +1,70 @@
...
+Initiate integrations with popular cloud platforms such as Azure IoT Hub, AWS IoT Core or nRF Cloud. 
+
+→ **Learn more about [**Cloud Integrations**](/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration/).**
+
+![Data Integrations](images/integrations.png "Data Integrations")
+
+
+## Device Test Repository*
+
+Test the performance of your device’s firmware by mimicking real-world scenarios. Device Tests allow you to validate new application firmware, to test the implementation of the LwM2M standard, and to run regressions tests to ensure new firmware doesn’t break existing functionality.
```
Comment (malw4) [inline docs/Coiote_IoT_DM/introduction.md:56]:
```suggestion
Test the performance of your device’s firmware by mimicking real-world scenarios. Device Tests allow you to validate new application firmware, to test the implementation of the LwM2M standard, and to run regression tests to ensure new firmware doesn’t break existing functionality.
```

File: docs/Coiote_IoT_DM/Visualize_data.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+---
+title: Data Visualization
+date: 2023-05-02
+og_title: AVSystem IoT Developer Zone
+---
+
+# Visualize device data
+
+The **Dashboard** consists of **widgets** which display telemetry data. In order to collect data to display, **observations** needs to be set on specific **resources**.
```
Comment (mikegpl) [inline docs/Coiote_IoT_DM/Visualize_data.md:9]: This is not entirely true. Right now, Coiote collects the data that is later displayed on dashboard from multiple sources:
- from NOTIFY messages, when you set an observation on a given resource,
- from SEND messages, which happen once device decides to send such message for a given resource; Coiote cannot directly trigger such SEND,
- from READs executed on device, for example during refresh datamodel

So I guess I would reword it to 
```
In order to collect sample data to display, one of following needs to happen:
- **observation** is set on a resource and device sends notifications with the resource value,
- **read** is invoked on either a resource or entire device data model,
- device itself sends a LwM2M SEND message with the updated resource value.
```

File: docs/Coiote_IoT_DM/introduction.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+## Dashboard
+
+The Dashboard consists of widgets which display telemetry data. Widgets use different kinds of charts depending on the type of data collected. Widgets are set and configured in the **Device Center**.
+
+→ **Learn more about [**Data Visualization**](/Coiote_IoT_DM/Visualize_data/).**
+
+## Operations Center
+
+Use the Operations Center to easily monitor your devices. It provides a clear overview with alerts, highlighting what devices require immediate attention.
```
Comment (mkrolavs) [inline docs/Coiote_IoT_DM/introduction.md:34]: put Operations center and any other names explicitly related to the UI in bold

**Review by mkrolavs — COMMENTED** [review]
comments for introduction.md and connect.md

File: docs/Coiote_IoT_DM/Visualize_data.md (hunk 3 above)
Comment (laurensslats) [reply docs/Coiote_IoT_DM/Visualize_data.md:9]: Thanks! Updated.

File: docs/Coiote_IoT_DM/introduction.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+## Operations center
+
+Use the **Operations center** to easily monitor your devices. It provides a clear overview of alerts, which highlight the devices requiring immediate attention.
+
+![Operations Center](images/operations-center.png "Operations Center")
+
+## Device groups*
+
+!!! Note
+    Features indicated marked with an "*****" are only included in the premium version of Coiote.
```
Comment (mkrolavs) [inline docs/Coiote_IoT_DM/introduction.md:46]: `indicated` to be removed

File: docs/Coiote_IoT_DM/introduction.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+!!! Tip "Sign up free of charge"
+    Get started by <a href="https://eu.iot.avsystem.cloud/" target="_blank">**signing up**</a> to Coiote IoT Device Management. You can sign up for a developer account which allows you to connect up to 10 devices free of charge.
+
+![Coiote login page](images/login-page.png "Coiote login page")
+
+## Device inventory
+
+Once logged in, you enter the **Device inventory** which provides an overview of all registered LwM2M devices. After selecting one of your devices, you enter the **Device center** which allows you to monitor and manage single devices. 
+
+In the Device center you can:
```
Comment (mkrolavs) [inline docs/Coiote_IoT_DM/introduction.md:21]: Device center in bold

**Review by mkrolavs — COMMENTED** [review]
minor issues

**Review by mkrolavs — APPROVED** [review]
